Makefile

```diff
@@ -385,7 +385,20 @@ function await() {
         exit $${RETCODE}
     fi
 }
-CMD="curl --fail --location --max-redir 0 --silent http://127.0.0.1/"
+
+# Some security settings require HTTP and others require HTTPS.
+# Therefore we test both HTTP and HTTPS.
+CURL_CMD="curl \
+	--insecure \
+	--fail \
+	--location \
+	--max-redir 0 \
+	--silent "
+HTTP_ADDRESS="http://127.0.0.1/"
+HTTPS_ADDRESS="https://127.0.0.1/"
+HTTP_CMD=$${CURL_CMD}$${HTTP_ADDRESS}
+HTTPS_CMD=$${CURL_CMD}$${HTTPS_ADDRESS}
+CMD="eval "$${HTTP_CMD}" || "$${HTTPS_CMD}
```

Comment:
> It looks like you may need to escape the quotation marks

Why? This seems to work.
In fact, the quotation marks aren't around `$${...}`s. They are around `"eval "` and `"  ||  "`.
This is because `bash` does string concatenation in a weird way.
Consider:

```
GREETING="hello"
NAME="Gustav"
NO_SPACE=$GREETING$NAME
BUT_WE_WANT_A_SPACE=$GREETING", "$NAME
```

Is there a way you'd like this to change to be clearer?

I'm wondering if I can make this mode clear.